queries.sql

```diff
@@ -16,3 +16,35 @@ SELECT * from animals WHERE name != 'Gabumon';
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+-- day 2
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+ROLLBACK;
+
+BEGIN
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+COMMIT;
+
+BEGIN
+DELETE FROM animals;
+ROLLBACK;
+SELECT * FROM animals;
+
+BEGIN;
+DELETE FROM animals WHERE date_of_birth > '2022-01-01';
+SAVEPOINT svp1;
+UPDATE animals SET weight_kg = weight_kg * -1;
+ROLLBACK TO SAVEPOINT svp1;
+UPDATE animals SET weight_kg = weight_kg * -1 WHERE weight_kg < 0;
+COMMIT;
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT neutered, MAX(escape_attempts) FROM animals GROUP BY neutered;
+SELECT MAX(weight_kg), MIN(weight_kg) FROM animals;
+SELECT species, AVG(escape_attempts) FROM animals
+WHERE date_of_birth BETWEEN '01-01-1990' AND '01-01-2000';
```

Comment:
- [ ] Same here for this query too. please add `GROUP BY species` to your query to make it right.
It will be like that:
```sql
SELECT species, AVG(escape_attempts) FROM animals GROUP BY species
WHERE date_of_birth BETWEEN '01-01-1990' AND '31-12-2000';
```